google/cloud/forseti/notifier/notifier.py

```diff
@@ -152,12 +154,19 @@ def run(inventory_index_id,
                             notifier['name'], resource['resource']))
                     progress_queue.put(log_message)
                     LOGGER.info(log_message)
-                    chosen_pipeline = find_notifiers(notifier['name'])
-                    notifiers.append(chosen_pipeline(
-                        resource['resource'], inventory_index_id,
-                        violation_map[resource['resource']], global_configs,
-                        notifier_configs, notifier.get('configuration')))
-
+                    try:
+                        chosen_pipeline = find_notifiers(notifier['name'])
+                        notifiers.append(chosen_pipeline(
+                            resource['resource'], inventory_index_id,
+                            violation_map[resource['resource']], global_configs,
+                            notifier_configs, notifier.get('configuration')))
+                    except Exception as e:  # pylint: disable=broad-except
+                        progress_queue.put('Error running \'{}\' notifier for '
```

Comment:
nit: For easier readability, can you move the string formatting out?

```
log_message = 'Error running \'{}\' notifier for resource \'{}\':  \'{}\''.format(
    notifier['name'], 
    resource['resource'], 
    traceback.format_exc()))
progress_queue.put(log_message)
```

Reply:
Fixed.